WORKSPACE

```diff
@@ -35,3 +35,26 @@ new_local_repository(
     path = "/usr/include/python3.6",  # May be overwritten by setup.py.
 )
 
+http_archive(
```

Comment:
i'm curious why we don't need to bring in rules_python for the //bindings/python/benchmark/... package that also uses py_library and py_binary. Any idea?

Reply:
right! these are basic python rules, no need to load anything

Reply:
just to be clear: rules_python are used for pip requirements management. can't see any other (simpler) ways of listing numpy and scipy dependencies for gbench

Reply:
that's fine. i'm trying to make sure we don't end up in some weird state where the requirements for the bindings and the requirements for the tools are somehow different and contradictory.